assets/stylesheets/_components.scss

```diff
@@ -41,5 +41,5 @@
 @import 'layout/community-translator/style';
 @import 'layout/sidebar/style';
 @import 'lib/preferences-helper/style';
-@import 'me/sidebar-navigation/style';
+@import 'my-sites/plan-features/style';
```

Comment:
Assuming this is unintentional :)

https://github.com/Automattic/wp-calypso/blob/97520a15af0c519fd62f71da251df09161a302b1/client/my-sites/plan-features/index.jsx#L73

Reply:
Good catch! Thanks :)